cmd/policy-controller/controller.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+
+	clientset "github.com/nmstate/k8s-node-net-conf/pkg/client/clientset/versioned"
+	informers "github.com/nmstate/k8s-node-net-conf/pkg/client/informers/externalversions/nmstate.io/v1"
+	listers "github.com/nmstate/k8s-node-net-conf/pkg/client/listers/nmstate.io/v1"
+)
+
+// Controller is the controller implementation for NodeNetConfPolicy resources
+type Controller struct {
+	// kubeclientset is a standard kubernetes clientset
+	kubeclientset kubernetes.Interface
+	// nmstateclientset is a clientset for our own API group
+	nmstateclientset clientset.Interface
+
+	policyLister listers.NodeNetConfPolicyLister
+	policySynced cache.InformerSynced
+
+	// workqueue is a rate limited work queue. This is used to queue work to be
+	// processed instead of performing it as soon as a change happens. This
+	// means we can ensure we only process a fixed amount of resources at a
+	// time, and makes it easy to ensure we are never processing the same item
+	// simultaneously in two different workers.
+	workqueue workqueue.RateLimitingInterface
+	// recorder is an event recorder for recording Event resources to the
+	// Kubernetes API.
+	recorder record.EventRecorder
+}
+
+// NewController returns a new state controller
+func NewController(
+	kubeclientset kubernetes.Interface,
+	nmstateclientset clientset.Interface,
+	policyInformer informers.NodeNetConfPolicyInformer) *Controller {
+	return nil
```

Comment:
why does it take any attributes if it returns nil anyway?

Reply:
this is not implemented yet. only state controller implemented